### PR TITLE
Finish multy level cate

### PR DESCRIPTION
**Pull request by upkey1010**

### CODE REVIEW

**Comment by upkey1010** [comment]
ready

File: app/views/categories/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= form_for @category do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label :parent_id %>
+  <%= f.collection_select :parent_id, @categories, :id, :name, {promt: t("choose_cate"), include_blank: "Super Category"} %>
```
Comment (dothidiemthao) [inline app/views/categories/_form.html.erb]: "Super Category" i18n

**Comment by dothidiemthao** [comment]
commented

**Comment by upkey1010** [comment]
ready

**Pull request merged**